src/test/java/com/coherent/task40/DownloadTest.java

```diff
@@ -0,0 +1,44 @@
+package com.coherent.task40;
+
+import org.junit.Before;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import static com.coherent.task40.TestConstants.*;
+
+public class DownloadTest {
+    WebDriver driver = new ChromeDriver();
+
+    @BeforeEach
+    void setup() {
+        BaseTest.setup(driver, DOWNLOAD_USER_PAGE);
+    }
+
+    @Test
+    public void download50Percentage() {
+        By downloadButton = By.id("cricle-btn");
+        By percentText = By.cssSelector(".percenttext");
+        String expectedPercent = "50|[51-100]%";
```

Comment:
I don't see where variable is used. Please remove it in case if you don't need it